src/xcoobee/sdk/Users.js

```diff
@@ -165,6 +166,42 @@ class Users {
     }
   }
 
+  /**
+   * Fetches the user's pgp public key.
+   *
+   * @async
+   * @param {string} xid - User's xcoobee ID.
+   * @param {Config} [config] - The configuration to use instead of the default.
+   *
+   * @returns {Promise<SuccessResponse, ErrorResponse>}
+   * @property {number} code - The response status code.
+   * @property {Error} [error] - The response error if status is not successful.
+   * @property {string} [error.message] - The error message.
+   * @property {string} request_id - The ID of the request generated by the XcooBee
+   *   system.
+   * @property {Object} [result] - The result of the response if status is successful.
+   * @property {Note} result.data - Some information on the user.

```

Comment:
We are returning just the public key -- not an object with a cursor, pgp_public_key, or xcoobee_id. Please update JSDocs.